ArduCopter/Parameters.cpp

```diff
@@ -1133,7 +1133,7 @@ const AP_Param::GroupInfo ParametersG2::var_info[] = {
     // @Range: -0.5 1.0
     // @User: Advanced
 
-    // @Param: PILOT_Y_RATE_TC
+    // @Param: PILOT_Y_RAT_TC
```

Comment:
could instead do Q_PLT_Y_RATE_TC in quadplane to avoid the copter change
i'd like an opinion from @rmackay9

Reply:
Changed to `_PLT_Y_` , copter changes dropped